File: igor/Problema0088/problema0088.java
import java.util.Scanner;
public class problema0088 {

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("escreve um numero: ");
        int x = input.nextInt();
    
        if(x>0){
            System.out.println("O numero é positivo");
        }else if(x<0){
            System.out.println("O numero é negativo");
        }else{
            System.out.println("O numero é zero");
        }

    }
}

File: igor/Problema0088/problema0088.js
const prompt = require("prompt-sync")({ sigint: true });

var x = prompt("Escreve um numero: ")

if(x>0){
    console.log("O numero é positivo")
}else if(x<0){
    console.log("O numero é negativo")
}else{
    console.log("O numero é zero")
}